2022-08-07-qubes-os-summit-history.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: "Qubes OS Summit: History from organizer's perspective"
+categories: talks
+---
+
+The next [Qubes OS Summit 2022](https://www.qubes-os.org/news/2022/07/29/qubes-os-summit-2022/) edition is upcoming. This year it will be held in Berlin from September 9th to 11th in hybrid format, in person and live-streamed for remote access. More details [here](https://qubesos.3mdeb.com/). Don't miss the event and more importantly how it started. In the article [Michał Żygowski from 3mdeb](https://blog.3mdeb.com/authors/michal-zygowski/) (Qubes OS Summit co-organizer) will describe the history and organizer's perspective of the event.
```

Comment:
Before this is published, I will be adding a short editor's note at the very top that incorporates some of this information so that readers understand that this is a guest post ([example](https://www.qubes-os.org/news/2022/04/10/windows-integration-by-tabit-pro/), so there is no need to self-preface. You can instead simply start at the beginning of your topic, if you prefer.

Reply:
Moved to a separate section.